### PR TITLE
Added web services changes

### PR DESCRIPTION
**Pull request by psav**

### CODE REVIEW

**Comment by psav** [comment]
To be merged after - https://github.com/RedHatInsights/clowder/pull/185

**Comment by kylape** [comment]
`internal` is still referenced in this PR

**Review by kylape — CHANGES_REQUESTED** [review]
`internal` is still referenced in this PR

File: index.js [hunk 1]
```diff
@@ -70,6 +70,21 @@ class Config {
         }
         return dependencyEndpoints;
     }
+
+    PrivateDependencyEndpoints() {
+        var privateDependencyEndpoints = new(Map);
+        var i;
+        if (Config.data.privateEndpoints){
+            for(i = 0; i < Config.data.privateEndpoints.length; i++){
```
Comment (kylape) [inline index.js:78]: would `Array.prototype.forEach()` work here?

**Review by kylape — CHANGES_REQUESTED** [review]
So do you now generate the code based on the schema?

**Pull request merged**